src/main/java/com/conveyal/datatools/manager/auth/Auth0Connection.java

```diff
@@ -132,8 +135,8 @@ private static void remapTokenValues(Map<String, Object> jwt) {
      * tables in the database.
      */
     public static void checkEditPrivileges(Request request) {
-        if (authDisabled()) {
-            // If in a development environment, skip privileges check.
+        if (authDisabled() || IN_TESTING_ENVIRONMENT) {
+            // If in a development or testing environment, skip privileges check.
```

Comment:
Why do we want to skip the edit privileges check here? Wouldn't it be valuable to check that the test user has editing privileges for a given feed source during the course of testing. Maybe there is a good reason, but it isn't obvious from the comment.

Reply:
Hmmm, that's not a bad point. Since this was the first draft of testing an endpoint that required authentication, I just disabled it checking all privileges entirely so that any endpoint requiring authentication could be tested. I suppose the more thorough way of doing this would be to send an actual token that verifies with an actual AUTH0_SECRET or AUTH0_PUBLIC_KEY config value. However, that could conceivably be accomplished by writing a unit test for this class that had bad keys and bad tokens and whatnot and found the proper errors. 

Perhaps the solution here would be to create an actual Auth0 tenant, get the keys for testing and then delete the tenant so that those keys then aren't able to do anything. Lmk if that sounds like a good plan.

Reply:
For now, I would just say that we should bulk up the comment with a little more detail on why we're skipping the check (maybe along with a `TODO`). I don't think we need to go down the path of creating and deleting an Auth0 tenant. My long-term, preferred approach might be to assign a more complex `Auth0UserProfile` to the `user` request attribute that has certain permissions that we want to test.